swagger.yaml

```diff
@@ -3666,6 +3738,10 @@ definitions:
         type: string
         description: JSON-formatted string containing the validation results for each validation id grouped by category (network, hosts-data, etc.)
         x-go-custom-tag: gorm:"type:varchar(2048)"
+      logs_info:
```

Comment:
What is the format of this string?

Reply:
currently it is filled from logs_state but we would like to keep it free-styled in case we would like to expand on it in the future

Reply:
I'd rather it be well-defined, just logs_state